mempool/mempool
# Save bisq aggregate exchange rates in the database for each new block

Context https://github.com/mempool/mempool/issues/1520. This PR only saves the exchanges rates, it does not consume the data yet.

Database migration version increased to 19.

* I ended up storing all fiat currencies (for a future currency switch feature) which are using `Bisq-Aggregate` provider as a JSON object in the database. We insert one new entry for each new block.
* Past blocks don't have historical price.
* We also do not insert prices unless we at least fetched them once (we don't insert zeroed prices).

### Testing

* Run the node backend with this PR
* Wait for a new block
* Run the following db query to confirm prices are properly inserted
```mysql
SELECT height, JSON_EXTRACT(bisq_rates, '$.USD') as USD,
JSON_EXTRACT(bisq_rates, '$.KRW') as KRW, bisq_rates FROM rates;
```
